portfolio/src/main/webapp/script.js

```diff
@@ -124,11 +125,15 @@ xhttp.onreadystatechange = function() {
       form.appendChild(br);
       form.appendChild(button);
       commentTitle.parentNode.insertBefore(form, commentTitle.nextSibling);
+
+      loginButton.value = "Logout";
     } else {
       var text = document.createElement('p');
```

Comment:
Same as above here in regards to 'var'.

Reply:
This is because the other PR has not been approved yet. I made the changes there and need them to put put into master